snippets/mcs-endpoint-limitation.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * modules/installation-about-custom-azure-vnet.adoc
+// * modules/machine-config-operator.adoc
+// * security/certificate_types_descriptions/machine-config-operator-certificates.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+There is no supported way to block or restrict the Machine Config Server (MCS) endpoint in the current architecture. The MCS must be exposed to the network so that newly-provisioned machines, which have no existing configuration or state, are able to fetch their configuration. In this model, the root of trust is the certificate signing requests (CSR) endpoint, which is where the kubelet sends its certificate signing request for approval to join the cluster. Because of this, machine configs should not be used to distribute sensitive information.
+
+To ensure that the MCS endpoints (accessible from ports 22623 and 22624) are secured in bare metal scenarios, customers must set-up proper network policies.
```

Comment:
Since we should be trying to avoid parentheses like this, I'd try to rewrite. Or at the very least cut it down to  to minimum.

Also backtick ports, and set-up should not be a hyphen since it's a verb. Or you could replace it with something like "configure".

```suggestion
To ensure that the MCS endpoints (ports `22623` and `22624`) are secured in bare metal scenarios, customers must set up proper network policies.
```